### PR TITLE
qa_crowbarsetup.sh: fix SLES12 repo setup

### PR DESCRIPTION
**Pull request by aspiers**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -539,42 +539,65 @@ function rsync_iso()
...
+}
 
-    # create empty repository when there is none yet
+# create empty repository when there is none yet
+function onadmin_create_sles12_repos()
+{
     safely zypper -n install createrepo
-    sles12optionalrepolist="SLE-12-Cloud-Compute5-Pool SLE-12-Cloud-Compute5-Updates SLE12-Cloud-Compute-PTF SLES12-Pool"
-    for repo in $sles12optionalrepolist ; do
+    sles12optionalrepolist=(
```
Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: should be local variable

Comment (aspiers) [inline scripts/qa_crowbarsetup.sh]: Thanks, fixed.

**Comment by jdsn** [comment]
:+1:  lgtm;
Did you have a test run with this patch?

**Comment by aspiers** [comment]
Yes.  It worked with the patch, and failed without it.

**Comment by vuntz** [comment]
This broke deployment of SLE12 nodes.

**Comment by aspiers** [comment]
Well it *unbroke* it for me ;-)  Before this PR, it did not deploy anything at all in local mode.  But this highlights the importance of https://trello.com/c/DvJ0H538 ...

The fix is apparently in #263.

**Pull request merged**